content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -7,6 +7,9 @@ content_type: concept
 weight: 20
 ---
 <!-- overview -->
+
+{{< feature-state for_k8s_version="v1.14" state="stable" >}}
```

Comment:
Hi @chenxuc .
What do you think about moving this feature-state shortcode to the 
`Pod's DNS Config` section (around line 267):

Reply:
Thanks for the review. I have moved the shortcode to the Pod's DNS Config section.